event-publisher/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "event-publisher"
```

Comment:
hmm since we are now using "event", should the existing mbus-api be renamed to event-api ? It seems to be specifically tailed to events?

Reply:
Yes. Also it does the work of mbus, so should it be called event-bus-api?

Reply:
Does it expose generic message bus api, or only for event messages?

Reply:
If it does also expose bus api I'd suggest perhaps `mbus-events`

Reply:
@sinhaashish @niladrih any suggestions?

Reply:
It exposes the publishing api only for event messages. Subscription is generic.

Reply:
Mbus api are used internally and eventing is a wrapper around this, Since we are using the eventing feature everywhere in our code base and mbus is used internally , we can rename it to events-api.

Reply:
Cool, then `mbus-events` would be my suggestion.

Reply:
Mbus api are used internally and eventing is a wrapper around this, Since we are using the eventing feature everywhere in our code base and mbus is used internally , we can rename it to events-api. -  from @sinhaashish